pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -759,6 +934,44 @@ func (lbaas *LbaasV2) EnsureLoadBalancerDeleted(clusterName string, service *api
 		return err
 	}
 	waitLoadbalancerDeleted(lbaas.network, loadbalancer.ID)
+
+	// Delete the Security Group
+	if lbaas.opts.ManageSecurityGroups {
+		// Generate Name
+		lbSecGroupName := getSecurityGroupName(clusterName, service)
+		lbSecGroupID, err := groups.IDFromName(lbaas.network, lbSecGroupName)
+
+		if err != nil {
+			// We were getting this group to delete it anyway - let just continue
+			glog.Errorf("Error occured finding securty group: %s", lbSecGroupName)
```

Comment:
typo: "securty"
Also: include `err` in the error message, otherwise we're losing valuable debugging information.